contrib_bots/lib/giphy/zulip_giphy_config.py

```diff
@@ -0,0 +1,2 @@
+# Change these values to configure authentication for the bot.
+GIPHY_API_KEY = 'dc6zaTOxFJmzC'
```

Comment:
oh, you're right, actually this is a public beta API key, thanks for reminding this.. !